phpunit.xml.dist

```diff
@@ -47,6 +47,7 @@
 
     <php>
         <ini name="zend.enable_gc" value="0"/>
+        <ini name="memory_limit" value="-1"/>
```

Comment:
I would except to see failure if our test would pass but used 20GB RAM.
so maybe some real value like 1GB or whatever is needed rather than no limit at all?

Reply:
Personally, I don't care about memory usage for cli tools.

Maybe it's better to set a max memory usage per fixer and display an error message when the memory exceeds this threshold? This way you also directly know which test is consuming a lot of memory.

Currently this is already been done for slow tests:
`You should really fix these slow tests (>100ms)...`


This could be implemented in a different PR.

Reply:
Does such test listener exists? i.e. "memory usage listener" similar to https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.8/phpunit.xml.dist#L36 ?

I made a mistake a few PR's back were I blew up memory usage and this was only detected by the utest on Travis.
Therefore I agree with @keradus here, maybe better to put it on something realistic.

Reply:
This listener exists, but I am not sure how good it will work:
https://github.com/atrapalo/phpunit-memory-and-time-usage-listener

I'll change the config for now